shardingsphere-proxy/shardingsphere-proxy-backend/src/main/java/org/apache/shardingsphere/proxy/backend/context/ProxyContext.java

```diff
@@ -131,4 +131,25 @@ public StateContext getStateContext() {
         result.addAll(contextManager.getMetaDataContexts().getGlobalRuleMetaData().getRules());
         return result;
     }
+    
+    /**
+     * Set system property.
+     *
+     * @param key property key
+     * @param value property value
+     */
+    public void setSystemProperty(final String key, final String value) {
```

Comment:
for me, I prefer to extract these two methods into a higher level of util or something like that.en .... you decide that, this is not mandatory .

Reply:
Thank you @taojintianxia .
I also had this idea and discussed it yesterday, but we didn't find a suitable place in the end.